src/Tribe/Tickets.php

```diff
@@ -923,7 +927,31 @@ public function ajax_refresh_settings() {
 			do_action( 'tribe_tickets_ticket_add', $post_id, $ticket, $data );
 
 			// Pass the control to the child object
-			return $this->save_ticket( $post_id, $ticket, $data );
+			$save_ticket = $this->save_ticket( $post_id, $ticket, $data );
+
+			tribe( 'tickets.handler' )->toggle_manual_update_flag( false );
+
+			$post = get_post( $post_id );
+			if ( empty( $data['ticket_start_date'] ) ) {
+				// 30 min in seconds
+				$round = 1800;
```

Comment:
You know you can do `30 * MINUTE_IN_SECONDS` and save the comment, right?
https://codex.wordpress.org/Easier_Expression_of_Time_Constants

Reply:
I like using the constants whenever possible - for transparency (and to reduce the chance I typo a number ;) )